xbmc/guilib/guiinfo/MusicGUIInfo.cpp

```diff
@@ -313,6 +313,36 @@ bool CMusicGUIInfo::GetLabel(std::string& value, const CFileItem *item, int cont
         return true;
       }
       break;
+      case LISTITEM_BITRATE:
+      {
+        int BitRate = tag->GetBitRate();
+        if (BitRate > 0)
+        {
+          value = StringUtils::Format("%i", BitRate);
+          return true;
+        }
+        break;
+      }
+      case LISTITEM_SAMPLERATE:
+      {
+        int sampleRate = tag->GetSampleRate();
+        if (sampleRate > 0)
+        {
+          value = StringUtils::Format("%i", sampleRate);
+          return true;
+        }
+        break;
+      }
+      case LISTITEM_NO_OF_CHANNELS:
```

Comment:
Possibly, but there are various references to `channels` for pvr as well as `audio_channels` for the video stuff so I was trying to be clear that this is not the same.  Also, musicplayer_channels may well be returning the same data (should be) but one is generated by the player at runtime and one is stored in the db so could technically be different.  Unlikely, but possible.

Reply:
Hum.... I see your point but  "NO_OF_" doesn't really separate it from all the other kinds of channel either. I would like to tie it to MUSICPLAYER_CHANNELS, which even if it is slightly different in runtime is conceptulaly reporting the same thing.  It is coming from MusicInfoTag so maybe LISTITEM_MUSICCHANNELS or LISTITEM_SONGCHANNELS (since only a song will have it). Archaic GUI issues....